RELEASES.md

```diff
@@ -57,7 +57,21 @@ When the release branch is ready, a tag should be pushed with a name matching th
 
 ### Weekly Dev Release
 
-Ratify is configured to generate and publish dev build images based on schedule [here](https://github.com/deislabs/ratify/blob/main/.github/workflows/publish-package.yml#L8).  
+#### Publishing Guidelines
+- Ratify is configured to generate and publish dev build images based on the schedule [here](https://github.com/deislabs/ratify/blob/main/.github/workflows/publish-package.yml#L8). 
+- Contributors MUST select the `Helm Chart Change` option under the `Type of Change` section if there is ANY update to the helm chart that is required for proposed changes in PR.
+- Maintainers MUST manually trigger the "Publish Package" workflow after merging any PR that indicates `Helm Chart Change`
+  - Go to the `Actions` tab for the Ratify repository
+  - Select `publish-ghcr` option from list of workflows on left pane
+  - Select the `Run workflow` drop down on the right side above the list of action runs
+  - Choose `Branch: main`
+  - Select `Run workflow`
+- Process to Request an off-schedule dev build be published
+  - Submit a new feature request issue prefixed with `[Dev Build Request]`
+  - In the the `What this PR does / why we need it` section, briefly explain why an off schedule build is needed
```

Comment:
Our there heuristics that would help folks explain/justify why this off-cycle build is needed?

Reply:
I think one helpful use case would be if Ratify introduces a new feature that does not have an accompanying helm chart change. For example, for cosign, if we start to support bundle verification, which is not a CRD level change but a specific verifier feature addition that would only impact the underlying code implementation, and someone wants to try that out immediately without having to wait for the next build. There are ways to build off main branch but I think this would get it in the hands of someone faster.